charts/seed-monitoring/charts/grafana/dashboards/operators/vpn-seed-server/vpn-seed-server-dashboard.json

```
@@ -0,0 +1,1188 @@
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "description": "OpenVPN Server status using Prometheus and OpenVPN exporter (High availability only)",
+  "editable": true,
+  "gnetId": null,
+  "graphTooltip": 0,
+  "links": [],
+  "panels": [
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": null,
+      "description": "Traffic for all VPN tunnels of all VPN clients of all all Kube-apiservers pods",
+      "fieldConfig": {
+        "defaults": {},
+        "overrides": []
+      },
+      "fill": 1,
+      "fillGradient": 0,
+      "gridPos": {
+        "h": 6,
+        "w": 24,
+        "x": 0,
+        "y": 0
+      },
+      "hiddenSeries": false,
+      "id": 26,
+      "interval": "1m",
+      "legend": {
+        "avg": false,
+        "current": false,
+        "max": false,
+        "min": false,
+        "show": true,
+        "total": false,
+        "values": false
+      },
+      "lines": true,
+      "linewidth": 1,
+      "links": [],
+      "nullPointMode": "null",
+      "options": {
+        "alertThreshold": true
+      },
+      "percentage": false,
+      "pluginVersion": "7.5.17",
+      "pointradius": 2,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "exemplar": true,
+          "expr": "sum (label_replace(rate(openvpn_server_client_received_bytes_total{common_name=~\"vpn-seed-client\"}[$__rate_interval]), \"service\", \"$1\", \"instance\", \"([^.]+)(.+)\")) by (service)",
+          "format": "time_series",
+          "interval": "1m",
+          "intervalFactor": 1,
+          "legendFormat": "{{service}} Received",
+          "refId": "A"
+        },
+        {
+          "exemplar": true,
+          "expr": "sum (label_replace(rate(openvpn_server_client_sent_bytes_total{common_name=~\"vpn-seed-client\"}[$__rate_interval]), \"service\", \"$1\", \"instance\", \"([^.]+)(.+)\")) by (service)",
+          "format": "time_series",
+          "hide": false,
+          "interval": "1m",
+          "legendFormat": "{{service}} Sent",
+          "refId": "B"
+        }
+      ],
+      "thresholds": [],
+      "timeFrom": null,
+      "timeRegions": [],
+      "timeShift": null,
+      "title": "Traffic VPN Clients of all Kube-apiservers (vpn-seed-client)",
+      "tooltip": {
+        "shared": true,
+        "sort": 0,
+        "value_type": "individual"
+      },
+      "type": "graph",
+      "xaxis": {
+        "buckets": null,
+        "mode": "time",
+        "name": null,
+        "show": true,
+        "values": []
+      },
+      "yaxes": [
+        {
+          "$$hashKey": "object:118",
+          "format": "decbytes",
```

```suggestion
          "format": "binBps",
```
Minor: use the `kB/s` unit for the traffic rate instead of `kB`. Note: there are a few other panels that could be adjusted similarly as well.

---

fixed